build/resolver/http/entrypoint.sh

```diff
@@ -65,10 +71,17 @@ URL=${URL//'${WORKFLOWRUN_NAME}'/${WORKFLOWRUN_NAME}}
 wrapPush() {
     if [ -z ${COMPRESS_FILE_NAME} ]; then COMPRESS_FILE_NAME="artifact.tar"; fi
     if [ -z ${FORM_FILE_KEY} ]; then FORM_FILE_KEY="file"; fi
+    if [ -z ${FIND_OPTIONS} ]; then FIND_OPTIONS=". -name '*'"; fi
 
-    echo "Start to compress files under ${WORKDIR}/data into file ${COMPRESS_FILE_NAME}"
-    cd ${WORKDIR}/data
-    tar -cvf ${WORKDIR}/${COMPRESS_FILE_NAME} *
+    cd ${WORKDIR}/data/${DATA_SUBDIR}
```

Comment:
This `cd` command is not necessary?

Reply:
It is necessary that we need to find and cp files in the ${DATA_SUBDIR}